packages/livechat/src/routes/Chat/component.js

```diff
@@ -217,19 +235,26 @@ class Chat extends Component {
 								this.notifyEmojiSelect = click;
 							}}
 							handleEmojiClick={this.handleEmojiClick}
+							isRecording={this.state.isRecording}
+							handleRecording={this.handleRecording}
 							pre={
 								<ComposerActions>
-									<ComposerAction className={createClassName(styles, 'emoji-picker-icon')} onClick={this.toggleEmojiPickerState}>
+									<ComposerAction isDisabled={this.state.isRecording} onClick={this.toggleEmojiPickerState}>
 										<EmojiIcon width={20} height={20} />
 									</ComposerAction>
 								</ComposerActions>
 							}
 							post={
 								<ComposerActions>
 									{text.length === 0 && uploads && (
-										<ComposerAction onClick={this.handleUploadClick}>
-											<PlusIcon width={20} height={20} />
-										</ComposerAction>
+										<ComposerActions>
+											<ComposerAction onClick={this.handleRecording} isDisabled={this.state.isRecording}>
```

Comment:
We need a new setting to enable/disable this feature, as it could be disrupting for existing users